.travis.yml

```diff
@@ -2,13 +2,15 @@ language: ruby
 cache: bundler
 before_install:
   - gem list bundler
+  - sudo pip install pycodestyle
 rvm:
   - "2.2.3"
   - "2.3.1"
   - "2.4.3"
   - "2.5.0"
 script:
   - 'bundle exec rubocop -D'
+  - 'pycodestyle --first scripts/'
```

Comment:
why using `first` option?

Reply:
`first` option shows first occurrence of each error, so we get the overall idea of what are the wrong parts of code while looking at travis logs. It also saves total execution time.

That's why I thought to use it.

Reply:
this is fine for travis, but for the rake task I would suggest rendering everything.